ydb/library/yql/minikql/mkql_program_builder.cpp

```diff
@@ -84,6 +84,14 @@ void EnsureScriptSpecificTypes(
         case EScriptType::CustomPython:
         case EScriptType::CustomPython2:
         case EScriptType::CustomPython3:
+        case EScriptType::SystemPython2:
+        case EScriptType::SystemPython3:
+        case EScriptType::SystemPython3_8:
```

Comment:
Out of interest, why did you choose 3.8 to be minimum supported version?

Reply:
There are 2 reasons:
1. Based on this picture
![Screenshot 2024-07-10 at 18 03 12](https://github.com/ydb-platform/ydb/assets/14953722/c36df848-7596-4e20-be2a-efe0e2217f3c)
https://devguide.python.org/versions/

2. Seems like python3.7 was not available as apt package on ubuntu 20.04 where I was developing it

Reply:
Ok, I see